plugins/omemo/src/ui/contact_details_dialog.vala

```diff
@@ -81,7 +81,7 @@ public class ContactDetailsDialog : Gtk.Dialog {
             own = true;
             own_id = plugin.db.identity.row_with(plugin.db.identity.account_id, account.id)[plugin.db.identity.device_id];
 
-            automatically_accept_new_descr.label = _("When you add new encryption keys to your account, automatically accept them.");
```

Comment:
Here, it was confusing that "you" first means the user and the imperative then addresses the software. I changed it to "I", so it should be clear that the user is telling the software to do something.